pytest_minio_mock/plugin.py

```diff
@@ -322,11 +333,35 @@ def get_object(
         """
         self._health_check()
         the_object = self.buckets[bucket_name][object_name]
+        if (
+                self.get_bucket_versioning(bucket_name).status == ENABLED
+                and isinstance(the_object, dict)
+        ):
+            # versioning is activated on the bucket, and the object is versioned
+            if not version_id:
+                version_id = list(the_object.keys())[-1]
+            try:
+                # TODO: should add version UUID validation, maybe with pydantic ?
```

Comment:
`validators` is used elsewhere in this package. For `uuid` check https://python-validators.github.io/validators/reference/uuid/

Reply:
done